aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/RealAWSCognitoAuthPluginTest.kt

```diff
@@ -472,4 +476,111 @@ class RealAWSCognitoAuthPluginTest {
 
         assertEquals(expectedException, resultCaptor.captured.cause)
     }
+
+    @Test
+    fun `update user attribute fails when not in SignedIn state`() {
```

Comment:
add the corresponding for update user attributes as well.

Reply:
done